search/tutorials/tutorial_pagination.md

```diff
@@ -1,155 +1,239 @@
-# **Pagination Tutorial**
+# Pagination tutorial
 
-## Let's get started
+## Get started
 
-Using pagination lets you view and navigate the search results effortlessly. Moreover, it decreases both the lookup time and the size of responses.
+Using pagination, you can view and navigate the search results effortlessly. Moreover, it decreases both the lookup time and the size of responses.
```

Comment:
Either: 
Using pagination lets you view and navigate the search results effortlessly.
OR 
You can view and navigate the search results effortlessly with pagination.